modules/apps/headless/headless-delivery/headless-delivery-test/src/testFunctional/macros/DocumentAPI.macro

```diff
@@ -225,14 +236,15 @@ definition {
 	macro getDocumentsByErcInAssetLibrary {
```

Comment:
Variables.assertDefined is missing here too, is it on purpose?

Reply:
maybe if we have loads of cases like this we should open another refactor task to review it? 🤔

Reply:
This is also on purpose because the only required variable is already defined in another method C, and the method B we called in the current method is called method C already. 
Really agree to create another refactor task to checking all the variables.assertDefined(), to check what we really need and I think maybe we can remove some of the variables.assertDefined.

Reply:
yeah true, however, this macro is directly connected with `.testcase` file and so we want to know from this layer what to put in order for the chain of macros work. I will create a new task to work on this